app/uk/gov/hmrc/cbcr/controllers/FileUploadResponseController.scala

```diff
@@ -38,8 +38,8 @@ class FileUploadResponseController @Inject()(repo: FileUploadRepository, auth: C
         error => Future.successful(BadRequest(JsError.toJson(error))),
         response =>
           repo.save2(response).map {
-            case result if result.ok => Ok
-            case result              => InternalServerError(result.writeErrors.mkString)
+            case result if result.wasAcknowledged() => Ok
+            case _                                  => InternalServerError("Mongo error")
```

Comment:
You are using this `("Mongo error")` message at couple of places. 
Is it possible to have more useful message than just generic message?

Reply:
This check apparently is redundant (with default write concern) according to the Mongo team - 500/error log will happen automatically.

Reference: https://www.mongodb.com/docs/manual/reference/method/db.collection.updateMany/#returns

I should probably go through the code and remove these checks, though.